tests/api/test_collect_runs_logs.py

```diff
@@ -166,6 +168,163 @@ async def test_collect_logs_on_startup(
             == {}
         )
 
+    @pytest.mark.asyncio
+    async def test_collect_logs_for_old_runs_on_startup(
+        self,
+        db: sqlalchemy.orm.session.Session,
+        client: fastapi.testclient.TestClient,
+    ):
+        log_collector = mlrun.api.utils.clients.log_collector.LogCollectorClient()
+
+        project_name = "some-project"
+        new_uid = "new_uid"
+        old_uid = "old_uid"
+
+        # create first run
+        _create_new_run(
+            db,
+            project_name,
+            uid=old_uid,
+            name=old_uid,
+            kind="job",
+            state=mlrun.runtimes.constants.RunStates.completed,
+        )
+
+        # sleep for 5 seconds to make sure the runs are not created at the same time
+        time.sleep(5)
```

Comment:
is it possible to manually change the creation time instead of sleeping? our unit tests are long as they are.

Reply:
Already looked for a way to do it and couldn't find it..
We can't actually access to the db from the test, and DBInterface doesn't have an option to set the creation time.